ipatests/prci_definitions/nightly_latest.yaml

```diff
@@ -1341,6 +1341,18 @@ jobs:
         timeout: 4800
         topology: *master_1repl
 
+  fedora-latest/test_ipahealthcheck:
```

Comment:
test_ipahealthcheck is already present in line 1332-1342, please update the test definition instead of adding a new one. The same comment applies to all the yaml files.

Reply:
Don't keep the same name for 2 different tests, otherwise they won't get executed at all.